.gitignore

```diff
@@ -27,4 +27,7 @@ credentials.json
 client_secret_*.json
 
 # Skip token.json
-token.json
\ No newline at end of file
+token.json
+
+# Downloaded files
+exports/
```

Comment:
Related to my other comment: considering that we are adding the folder to `gitignore`, it makes sense that the `exports/` folder don't be available on git history.

Reply:
alright I'll make the changes